meson.build

```diff
@@ -1897,7 +1897,7 @@ if 'auto' in init_style
     endif
 endif
 
-if init_cmd != ''
+if init_cmd != '' and get_option('with-init-hooks') == true
```

Comment:
Can you please remove the `== true` from the condition? We consistently use the implicit boolean comparison in all the meson scripts.